### ISSUE TITLE
NPE is logged when trying to open Eclipse perspective customization

### ISSUE THREAD

**Issue by allati** [dbeaver/dbeaver#5861]
#### System information: 
- DBeaver version 6.0.4.201905052017
- Java version OpenJDK Runtime Environment (build 12+33)
- Eclipse 2019-06 (4.12, build id I20190411-0625) with lots of extensions, DBeaver included

#### Describe the problem you're observing:
Since 6.0.4 upgrade when I try to open perspective customization (Window ->Perspective -> Customize perspective...) no window is opened and error is logged instead:

```
org.eclipse.e4.core.di.InjectionException: java.lang.NullPointerException
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:68)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invokeUsingClass(InjectorImpl.java:320)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invoke(InjectorImpl.java:254)
	at org.eclipse.e4.core.contexts.ContextInjectionFactory.invoke(ContextInjectionFactory.java:173)
	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:156)
	at org.eclipse.core.commands.Command.executeWithChecks(Command.java:497)
	at org.eclipse.core.commands.ParameterizedCommand.executeWithChecks(ParameterizedCommand.java:490)
	at org.eclipse.e4.core.commands.internal.HandlerServiceImpl.executeHandler(HandlerServiceImpl.java:213)
	at org.eclipse.ui.internal.handlers.LegacyHandlerService.executeCommand(LegacyHandlerService.java:403)
	at org.eclipse.ui.internal.actions.CommandAction.runWithEvent(CommandAction.java:154)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:568)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:400)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4131)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1055)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:3944)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3547)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1449)
Caused by: java.lang.NullPointerException
	at org.jkiss.dbeaver.ui.actions.datasource.SelectActiveDataSourceHandler.updateElement(SelectActiveDataSourceHandler.java:98)
	at org.eclipse.ui.internal.handlers.HandlerProxy.updateElement(HandlerProxy.java:454)
	at org.eclipse.ui.internal.handlers.E4HandlerProxy.updateElement(E4HandlerProxy.java:124)
	at org.eclipse.ui.internal.WorkbenchHandlerServiceHandler.updateElement(WorkbenchHandlerServiceHandler.java:46)
	at org.eclipse.ui.internal.commands.CommandService.registerElement(CommandService.java:332)
	at org.eclipse.ui.internal.commands.CommandService.registerElementForCommand(CommandService.java:312)
	at org.eclipse.ui.internal.commands.CommandService.registerElementForUpdate(CommandService.java:398)
	at org.eclipse.e4.ui.workbench.renderers.swt.HandledContributionItem.postMenuFill(HandledContributionItem.java:198)
	at org.eclipse.e4.ui.workbench.renderers.swt.AbstractContributionItem.fill(AbstractContributionItem.java:266)
	at org.eclipse.jface.action.MenuManager.doItemFill(MenuManager.java:731)
	at org.eclipse.jface.action.MenuManager.update(MenuManager.java:813)
	at org.eclipse.jface.action.MenuManager.updateAll(MenuManager.java:935)
	at org.eclipse.jface.action.MenuManager.update(MenuManager.java:833)
	at org.eclipse.jface.action.MenuManager.updateAll(MenuManager.java:935)
	at org.eclipse.ui.internal.dialogs.cpd.CustomizePerspectiveDialog.loadMenuAndToolbarStructure(CustomizePerspectiveDialog.java:1648)
	at org.eclipse.ui.internal.dialogs.cpd.CustomizePerspectiveDialog.<init>(CustomizePerspectiveDialog.java:536)
	at org.eclipse.ui.internal.WorkbenchWindow.createCustomizePerspectiveDialog(WorkbenchWindow.java:3186)
	at org.eclipse.ui.internal.WorkbenchPage.editActionSets(WorkbenchPage.java:1990)
	at org.eclipse.ui.internal.handlers.EditActionSetsHandler.execute(EditActionSetsHandler.java:40)
	at org.eclipse.ui.internal.handlers.HandlerProxy.execute(HandlerProxy.java:294)
	at org.eclipse.ui.internal.handlers.E4HandlerProxy.execute(E4HandlerProxy.java:96)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:58)
	... 38 more
```
There was no such error in 6.0.3: I was trying use perspective customization to remove DBeaver UI contributions from toolbar of "Java EE" perspective.

**Comment by serge-rider** [comment]
This issue is already fixed in Early Access version. We'll publish new DBeaver update along with 6.0.5 version.

**Comment by allati** [comment]
Hi, @serge-rider!

I was going to verify that the issue is fixed now that 6.0.5 is released, but I'm having trouble doing update from DBeaver update site. I've tried Eclipse 4.12 and pristine 4.10, built update site from the latest devel branch, results are the same (except for plugin version dates obviously).

Am I missing something? Can you give me a hint where to look?

Thanks!

```
Cannot complete the install because one or more required items could not be found.
  Software being installed: DBeaver IDE 6.0.5.201905192156 (org.jkiss.dbeaver.ide.feature.feature.group 6.0.5.201905192156)
  Missing requirement: DBeaver GIS Support 1.0.33.201905192156 (org.jkiss.dbeaver.data.gis 1.0.33.201905192156) requires 'osgi.bundle; org.jkiss.bundle.gis 0.0.0' but it could not be found
  Cannot satisfy dependency:
    From: DBeaver Databases 1.0.0.201905192156 (org.jkiss.dbeaver.db.feature.feature.group 1.0.0.201905192156)
    To: org.eclipse.equinox.p2.iu; org.jkiss.dbeaver.ext.mysql [2.1.94.201905192156,2.1.94.201905192156]
  Cannot satisfy dependency:
    From: DBeaver MySQL Support 2.1.94.201905192156 (org.jkiss.dbeaver.ext.mysql 2.1.94.201905192156)
    To: osgi.bundle; org.jkiss.dbeaver.data.gis 0.0.0
  Cannot satisfy dependency:
    From: DBeaver IDE 6.0.5.201905192156 (org.jkiss.dbeaver.ide.feature.feature.group 6.0.5.201905192156)
    To: org.eclipse.equinox.p2.iu; org.jkiss.dbeaver.db.feature.feature.group [1.0.0.201905192156,1.0.0.201905192156]
```
